src/index.html

```diff
@@ -11,7 +11,177 @@
+            name="name"
+            id="name"
+            autocomplete="off"
+            required
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label class="form-label" for="age">How old are You?
+            <input
+            type="number"
+            name="age"
+            id="age"
+            min="1"
+            max="100"
+            value="12"
+            required
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label class="form-label" for="dateOfBirth">Full date of birth:
+            <input
+            type="date"
+            name="dateOfBirth"
+            id="dateOfBirth"
+            required
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="terms">I accept the term of the agreement
+            <input
+            type="checkbox"
+            name="terms"
+            id="terms"
+            >
+          </label>
+        </div>
+
+      </fieldset>
+
+      <fieldset class="group">
+        <legend>Registration</legend>
+        <div class="form-field">
+          <label class="form-label" for="email">E-mail:
+            <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            required
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="password">Password:
+            <input
+            type="password"
+            name="password"
+            id="password"
+            minlength="5"
+            maxlength="15"
+            required
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="group">
+        <legend>An interesting fact about you!</legend>
+        <div>Do you love cats?
+          <label class="form-label" for="yes">
+            <input
+            type="radio"
+            name="cats"
+            id="yes"
+            > Yes
+          </label>
+
+          <label class="form-label" for="no">
+            <input
+            type="radio"
+            name="cats"
+            id="no"
+            > No
+          </label>
+        </div>
+
+        <label class="form-label own-line" for="color">What is your favorite color?
+          <input type="color" name="color" id="color">
+        </label>
+
+        <label class="form-label own-line" for="time">What time do you go to bed?
+          <input
+          type="time"
+          name="time"
+          id="time"
+          step="1"
+          >
+        </label>
+
+        <label class="form-label own-line" for="car-brand">What are your favorite brand of cars?
+          <select
+          name="car-brand"
+          id="car-brand"
+          multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
```

Comment:
please use lower-case for value attributes - it will save letters-case-driven headache when processing data on back-end.
```suggestion
            <option value="bmw">BMW</option>
            <option value="audi">Audi</option>
            <option value="lada">Lada</option>
```